src/canvas/Canvas.ts

```diff
@@ -1400,17 +1399,11 @@ export class Canvas extends SelectableCanvas implements CanvasOptions {
         target._findTargetCorner(this.getViewportPoint(e));
 
     if (!corner) {
-      if ((target as Group).subTargetCheck) {
-        // hoverCursor should come from top-most subTarget,
-        // so we walk the array backwards
```

Comment:
wrong

Reply:
What is wrong exactly?

Reply:
the order of targets is `[topMost, ..., bottomMost]`?

Reply:
> Sorted in descending z index order, top most object first